### PR TITLE
JavaScript capstone project submission

### PR DESCRIPTION
**Pull request by HERMON-1995**
The JavaScript capstone project is about building our own web application based on an external API.
We selected the [TVmaze API](https://www.tvmaze.com/api) that provides data about movies.
The webapp has 2 user interfaces:
   - The home page.
   - The comments popup.

We made use of the [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) to record likes and comments.
We also wrote functions to count the movies and comments displayed on the page.
We wrote tests for the counter functions.

Project Presentation video: [Link](https://drive.google.com/file/d/1bFXA9lbGRSvfPePjyld2wqQG3ucE9qgS/view?usp=sharing)

### CODE REVIEW

File: src/tests/movies-counter.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import countMovies from '../modules/movies-counter.js';
+
+describe('Test movie counter', () => {
```
Comment (huseyinbiyik) [inline src/tests/movies-counter.test.js:7]: You did a great job in general. And good job with using the jest testing library. Let's think at least about the scenario in which there are 0 items on the page (counter function should return 0). On the comments counter, you have that test but here not. Optionally, if you have any other edge case you can add it in your test. By testing edge cases like these, developers can ensure that their software works correctly and reliably under all possible scenarios. 👍

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @HERMON-1995 & @preciousbetine 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- You used ES6 and async functions 💯 
- You reviewed each other's PR very well 💯 
- You used separated modules for functions 💯 
- Design looks great and unique 💯 
- Linter set up well and checks pass 🟢
- Git flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- To fix the typo "invlovement.js" need to be "involvement.js". Software developers are usually detail-oriented, it is a more professional approach to keep each side of project is all correct 👍   

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @HERMON-1995 and @preciousbetine     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HERMON-1995** [comment]
Thank you so much for approving our project. We are thrilled to have the opportunity to move forward with this project and are grateful for your confidence in our team. 😊
